src/test/forms/inputs/ControlledRadioYesNo.test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { last } from 'lodash';
+
+import { ControlledRadioYesNo } from '../../../forms/inputs';
+
+const yesRadio = (control) => {
+  return control.find('Radio[label="Yes"]');
+};
+const noRadio = (control) => {
+  return control.find('Radio[label="No"]');
+};
+const selectYes = (control) => {
+  control.instance().handleChange({}, { value: 'Yes' });
+  control.update();
```

Comment:
I'm curious why this couldn't be written similar to `selectNo` (I'm sure there's a good reason...)

Reply:
I think this has to do with how semantic-ui-react changes the props of their input components in ways that aren't useful. I'm actually surprised that `selectNo()` is different than _this_. I'm curious about that instead.